test/integration/channel-transfer-multi/channel_transfer_multi_test.go

```diff
@@ -0,0 +1,495 @@
+
+		requestID := uuid.NewString()
+		nonce := client.NewNonceByTime().Get()
+		signArgs := append(append([]string{"channelMultiTransferByAdmin", requestID, cmn.ChannelIndustrial, cmn.ChannelIndustrial}, channelTransferArgs...), nonce)
+		publicKey, sign, err := admin.Sign(signArgs...)
+		Expect(err).NotTo(HaveOccurred())
+
+		transfer := &cligrpc.MultiTransferBeginAdminRequest{
+			Generals: &cligrpc.GeneralParams{
+				MethodName: "channelMultiTransferByAdmin",
+				RequestId:  requestID,
+				Chaincode:  cmn.ChannelIndustrial,
+				Channel:    cmn.ChannelIndustrial,
+				Nonce:      nonce,
+				PublicKey:  publicKey,
+				Sign:       base58.Encode(sign),
+			},
+			IdTransfer: channelTransferArgs[0],
+			ChannelTo:  channelTransferArgs[1],
+			Address:    channelTransferArgs[2],
+			Items:      transferItems,
+		}
+
+		By("sending multi transfer request")
+		r, err := apiClient.MultiTransferByAdmin(clientCtx, transfer)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(r.Status).To(Equal(cligrpc.TransferStatusResponse_STATUS_IN_PROCESS))
+
+		By("checking multi transfer status")
+		transferStatusRequest := &cligrpc.TransferStatusRequest{
+			IdTransfer: transferID,
+		}
+
+		excludeStatus := cligrpc.TransferStatusResponse_STATUS_IN_PROCESS.String()
+		value, err := anypb.New(wrapperspb.String(excludeStatus))
+		Expect(err).NotTo(HaveOccurred())
+
+		transferStatusRequest.Options = append(transferStatusRequest.Options, &typepb.Option{
+			Name:  "excludeStatus",
+			Value: value,
+		})
+
+		ctx, cancel := context.WithTimeout(clientCtx, network.EventuallyTimeout*2)
+		defer cancel()
+
+		By("awaiting for channel transfer to respond")
+		statusResponse, err := apiClient.TransferStatus(ctx, transferStatusRequest)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(statusResponse.Status).To(Equal(cligrpc.TransferStatusResponse_STATUS_COMPLETED))
+
+		By("checking result balances")
+		for i, expected := range expectedIndustrialBalances {
+			group := strings.Split(expected.GetToken(), "_")[1]
+
+			client.Query(network, peer, cmn.ChannelIndustrial, cmn.ChannelIndustrial,
+				fabricnetwork.CheckResult(fabricnetwork.CheckIndustrialBalance(group, expected.GetAmount()), nil),
+				"industrialBalanceOf", user.AddressBase58Check)
+
+			client.Query(network, peer, cmn.ChannelCC, cmn.ChannelCC,
+				fabricnetwork.CheckResult(fabricnetwork.CheckBalance(transferItems[i].GetAmount()), nil),
+				"allowedBalanceOf", user.AddressBase58Check, transferItems[i].GetToken())
+		}
+	})
+
+	It("multi transfer by customer test", func() {
+		By("creating grpc connection")
+		clientCtx = metadata.NewOutgoingContext(context.Background(), metadata.Pairs("authorization", networkFound.ChannelTransfer.AccessToken))
+
+		transportCredentials := insecure.NewCredentials()
+		grpcAddress := networkFound.ChannelTransfer.HostAddress + ":" + strconv.FormatUint(uint64(networkFound.ChannelTransfer.Ports[cmn.GrpcPort]), 10)
```

Comment:
It seems that there are functions in standard libraries that make address from host and port

Reply:
Function `net.JoinHostPort` used